teps/0101-env-in-pod-template.md

```diff
@@ -134,6 +136,10 @@ Besides, users quite frequently are willing to overwrite environment variable va
 specified in a `stepTemplate` in the single place when running pipelines.
 That helps to optionally pass settings, without a need to define additional pipeline parameters.
 
+Also, users can specify the global environment variables which needs to be added for every `TaskRun` or
+`PipelineRun`, then they not need to specify that every time in all `Tasks` and changing the values of that will also
```

Comment:
i think it may need some rephrasing, i am not sure if i understand the meaning without someone explaining it to me :)

Reply:
corrected